### PR TITLE
update README with Project Info

### PR DESCRIPTION
**Pull request by lfryc**
@edewit could you please review that I directed people right to Documentation and Issue Tracker?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,17 @@
 AeroGear Windows Cookbook
 =========================
 
-
 The AeroGear Windows cookbook is a list of recipes to quick start your Windows AeroGear experience.
 
+|                 | Project Info  |
+| --------------- | ------------- |
+| License:        | Apache License, Version 2.0  |
+| Build:          | Visual Studio  |
```
Comment (matzew) [inline README.md:9]: more a general question: is that the only "build tool" ? 

I know Apache NPanday :-) but that does not really make sense, IMO ...

Comment (lfryc) [inline README.md:9]: yea, 'Build Tool' might be better wording. @edewit ?

**Comment by lfryc** [comment]
I would like to reuse this as Project Info template for https://github.com/aerogear/aerogear-windows-oauth2 and https://github.com/aerogear/aerogear-windows-push

File: README.md [hunk 2]
```diff
@@ -1,9 +1,17 @@
...
 
-
 The AeroGear Windows cookbook is a list of recipes to quick start your Windows AeroGear experience.
 
+|                 | Project Info  |
+| --------------- | ------------- |
+| License:        | Apache License, Version 2.0  |
+| Build:          | Visual Studio  |
+| Documentation:  | https://aerogear.org/windows/  |
+| Issue tracker:  | [https://issues.jboss.org/browse/AEROGEAR/](https://issues.jboss.org/browse/AEROGEAR/component/12314947/?selectedTab=com.atlassian.jira.jira-projects-plugin:component-summary-panel)  |
```
Comment (lfryc) [inline README.md:11]: Let's wait for [AEROGEAR-1650](https://issues.jboss.org/browse/AEROGEAR-1650) in order to get Windows-specific issue tracker.

**Comment by danielpassos** [comment]
I'm closing it for now to prevent accidental merge

**Comment by matzew** [comment]
yeah, good call!

On Wed, Feb 25, 2015 at 12:51 PM, Daniel Passos notifications@github.com
wrote:

> Closed #3 https://github.com/aerogear/aerogear-windows-cookbook/pull/3.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/aerogear/aerogear-windows-cookbook/pull/3#event-241641421
> .

## 

Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf

**Pull request closed without merging**